src/getUserEmoji.js

```diff
@@ -120,24 +93,19 @@ function getEmojis(regex, text) {
   if (emojis === null) {
     return null
   }
-  let cleanedEmojis = []
-  emojis.forEach(emoji => {
-    let strippedEmoji = emoji.replace(/:/gi, '')
-    // remove skin tones, numbers and aws crap
-    if (
+  let cleanedEmojis = emojis.map(emoji => emoji.replace(/:/gi, '')).filter(
+    strippedEmoji =>
+      // remove skin tones, numbers and aws crap
       !excluded.includes(strippedEmoji) &&
       isNaN(strippedEmoji) &&
```

Comment:
This is a bit that seems a little funky - isNaN will - I think - always be true for the results of a match call, because, say "10" is NaN.